### DISCUSSION TITLE
[M-Z] (2024/25) - sezione 0.3 - es. 1 (7 ottobre)

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/calcolo-delle-probabilita#277] [Esercizi M-Z]
![1](https://github.com/user-attachments/assets/31bbecb3-8f2e-4eb0-98f9-fbfa748b93e4)
![2](https://github.com/user-attachments/assets/65ffa452-2edb-4f88-9780-c0d063fbc27c)

**Comment by marcoseverini** [comment]
Esercizio 9:
![immagine](https://github.com/user-attachments/assets/134944dc-4038-406d-a385-7fd52f22a1f5)

**Comment by marcoseverini** [comment]
Esercizio 10:
![immagine](https://github.com/user-attachments/assets/07de94c1-d2ef-48d2-a106-95fa92ba2736)

**Comment by marcoseverini** [comment]
Esercizio 12:
![immagine](https://github.com/user-attachments/assets/bb2a06e4-09ce-40c9-9032-68daba96776e)
![immagine](https://github.com/user-attachments/assets/0d566ee2-78db-42d3-89fe-67fc0ab6377d)
